### PR TITLE
game controller: fixed comments

### PR DESCRIPTION
**Pull request by SammyRamone**
this fixes https://github.com/bit-bots/humanoid_league_misc/issues/7
I didnt add much comments since the code is very straight forward and does not require much more documentation in my oppinion

### CODE REVIEW

File: humanoid_league_game_controller/src/humanoid_league_game_controller/receiver.py [hunk 1]
```diff
@@ -4,9 +4,7 @@
 from __future__ import unicode_literals, print_function
 
 """
-This module shows how the GameController Communication protocol can be used
-in python and also allows to be changed such that every team using python to
-interface with the GC can utilize the new protocol.
+This code provides a gamecontroller client fo the RoboCup Humanoid League.
```
Comment (NFiedler) [inline humanoid_league_game_controller/src/humanoid_league_game_controller/receiver.py]: for

File: humanoid_league_game_controller/src/humanoid_league_game_controller/receiver.py [hunk 2]
```diff
@@ -131,8 +129,8 @@ def answer_to_gamecontroller(self, peer):
             rospy.logerr("Network Error: %s" % str(e))
 
     def on_new_gamestate(self, state):
-        """ Is called with the new game state after receiving a package
-            Needs to be implemented or set
+        """ Is called with the new game state after receiving a package.
+            The information are processed and published as a standard message to a ROS topic.
```
Comment (NFiedler) [inline humanoid_league_game_controller/src/humanoid_league_game_controller/receiver.py]: information is processed

File: humanoid_league_game_controller/scripts/sim_gamestate.py [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 # This script was based on the teleop_twist_keyboard package
 # original code can be found at https://github.com/ros-teleop/teleop_twist_keyboard
+# The script provides a simple mechanism to test robot behaviour to different gamestates,
```
Comment (NFiedler) [inline humanoid_league_game_controller/scripts/sim_gamestate.py]: for different gamestates?

File: humanoid_league_game_controller/src/humanoid_league_game_controller/receiver.py (hunk 1 above)
Comment (jgueldenstein) [reply humanoid_league_game_controller/src/humanoid_league_game_controller/receiver.py]: fixed

File: humanoid_league_game_controller/scripts/sim_gamestate.py (hunk 3 above)
Comment (jgueldenstein) [reply humanoid_league_game_controller/scripts/sim_gamestate.py]: I think it should be "in different gamestates"

**Pull request merged**